frame/staking/src/pallet/mod.rs

```diff
@@ -141,6 +148,31 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominatorRewardedPerValidator: Get<u32>;
```

Comment:
Hmm, also it seems like you are using this wrong. 

We need two very distinct thing: 

One is the maximum number of `exposure.others` in `ErasStakers`. This has nothing to do with `MaxNominatorRewardedPerValidator` and should be called `MaxIndividualExposures`.

Then you have the same thing in `ErasStakersClipped`. This one should use `MaxNominatorRewardedPerValidator`. For consistency, if @thiolliere approves, I recommend naming these two more similar to one another, maybe calling the later `MaxRewardableIndividualExposures`.

Reply:
That is done, I have not renamed `MaxNominatorRewardedPerValidator` to `MaxRewardableIndividualExposures` yet though.

Reply:
Done with renaming.